### PR TITLE
Issue #939 Add method to handle single quote string to password

### PR DESCRIPTION
**Pull request by praveenkumar**
I hope this will patch up all special character war in password string.

### CODE REVIEW

File: cmd/minishift/cmd/start.go [hunk 1]
```diff
@@ -537,7 +537,7 @@ func ocSupportFlag(cmdName string, flag string) bool {
 
 func setSubcriptionManagerParameters() {
 	cluster.RegistrationParameters.Username = viper.GetString(username)
-	cluster.RegistrationParameters.Password = viper.GetString(password)
+	cluster.RegistrationParameters.Password = util.EscapeSingleQuote(viper.GetString(password))
```
Comment (hferentschik) [inline cmd/minishift/cmd/start.go]: IMO this should be done here - https://github.com/minishift/minishift/blob/088338ecc7071f56c869eb2ffe7ec78df722f734/pkg/minishift/registration/redhat.go#L60-L60. This is just needed for using the password as part of this command.

File: pkg/util/utils.go [hunk 2]
```diff
@@ -151,3 +151,8 @@ func GetHttpClient(proxyUrl string) *http.Client {
 	}
 	return myClient
 }
+
+func EscapeSingleQuote(s string) string {
+	r := strings.NewReplacer(`'`, `'"'"'`)
```
Comment (hferentschik) [inline pkg/util/utils.go]: +1

**Comment by praveenkumar** [comment]
> I take it you tested this end to end as well?

@hferentschik I did my best :)

**Comment by hferentschik** [comment]
> I did my best :)

LOL, fair enough

**Pull request merged**